### ISSUE TITLE
bug: Deleted agent with same email cannot be added again

### ISSUE THREAD

**Issue by nithindavid** [chatwoot/chatwoot#5747]
**Describe the bug**

If an agent is added, then deleted and later someone tries to add the same agent with the same email again, an "Agent email already in use, Please try another email address" message is displayed. The bug can be easily reproduced. It was possible to mitigate it by using different case letters.

**Comment by pranavrajs** [comment]
@nithindavid Can you please share a screencast? I cannot reproduce this on my end.

**Comment by Diplomat-LV** [comment]
@pranavrajs Please see the attached screenshot. Tested with Chatwoot v2.11.0

<img width="960" alt="Add_user" src="https://user-images.githubusercontent.com/118850916/203406889-ba61361e-0a5a-40ad-8ac6-cfae00577480.png">

**Comment by pranavrajs** [comment]
@tejaswinichile @sojan-official I figured out the bug here. It seems like if you use the downcased email, it would work without issues. 

@Diplomat-LV Can you please confirm if `test.user@testdomain.com` works instead of `Test.User@testdomain.com`?

**Comment by Diplomat-LV** [comment]
@pranavrajs That is correct. test.user@testdomain.com works and error message is only present if upcased letters are used.
